### PR TITLE
[.NET] move project files so apps are not nested

### PR DESCRIPTION
**Pull request by lucaspimentel**
## Motivation

Make local builds and debugging easier for .NET tests.

## Changes

Move the .NET project files so the `parametric` app is no longer nested inside the `weblog` app. This makes it easier to work with the projects locally, because the .NET SDK by default includes all files in a directory recursively.

This also simplifies the `Dockerfile` since we can copy the whole app directory without selecting each directory individually (we used to do this to exclude the nested `parametric` directory).

Bonus fix: some files also had their line endings fixed so they are saved with LF in GitHub. By default, when text files are checked out on Windows, git will convert the line endings to CRLF, and then back to LF when pushing commits. Before #2628, these files had UTF-8 byte-order marks (BOM) which prevents git from detecting them as text files. If you ignore whitespace changes in this PR, you can see there is no diff in any of the `*.cs` files.

## Workflow

1. ⚠️ Create your PR as draft ⚠️
2. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
3. Mark it as ready for review
    * Test logic is modified? -> Get a review from RFC owner. We're working on refining the `codeowners` file quickly.
    * Framework is modified, or non obvious usage of it -> get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

:rocket: Once your PR is reviewed, you can merge it!

🛟 [#apm-shared-testing](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X) 🛟

## Reviewer checklist

* [ ] [Relevant label](https://github.com/DataDog/system-tests/blob/main/docs/CI/labels.md) (`run-parametric-scenario`, `run-profiling-scenario`...) are presents
* [ ] If PR title starts with `[<language>]`, double-check that only `<language>` is impacted by the change
* [ ] No system-tests internal is modified. Otherwise, I have the approval from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)
* [ ] CI is green, or failing jobs are not related to this change (and you are 100% sure about this statement)
* [ ] A docker base image is modified?
    * [ ] the relevant `build-XXX-image` label is present
* [ ] A scenario is added (or removed)?
    * [ ] Get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

### CODE REVIEW

**Comment by lucaspimentel** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/DataDog/system-tests/2625?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#2625** <a href="https://app.graphite.dev/github/pr/DataDog/system-tests/2625?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#2624** <a href="https://app.graphite.dev/github/pr/DataDog/system-tests/2624?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @lucaspimentel and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: utils/build/docker/dotnet/weblog/app.sh
Comment (zacharycmontoya) [inline utils/build/docker/dotnet/weblog/app.sh:1]: Can you make sure that the permissions on the file stay the same? I think they are executable, so you can do the following to update them with `+x`: https://stackoverflow.com/questions/40978921/how-to-add-chmod-permissions-to-file-in-git

Comment (lucaspimentel) [reply utils/build/docker/dotnet/weblog/app.sh:1]: Good catch!
image

Comment (lucaspimentel) [reply utils/build/docker/dotnet/weblog/app.sh:1]: Fixed!

**Review by zacharycmontoya — APPROVED** [review]
LGTM, thanks!

**Comment by lucaspimentel** [comment]
### Merge activity

* **Jun 28, 11:04 AM EDT**: @lucaspimentel started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/DataDog/system-tests/2625).

**Pull request merged**